serverless/serverless-update.adoc

```diff
@@ -0,0 +1,18 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-update"]
+= Update
+include::modules/common-attributes.adoc[]
+:context: serverless-update
+
+toc::[]
+
+If you have installed a previous version of {ServerlessProductName}, you can follow the instructions in this guide to update to the latest version.
+
+[IMPORTANT]
+====
+Before updating to the latest {ServerlessProductName} release, you must remove the community Knative Eventing Operator if you have previously installed it.
+
+Having the Knative Eventing Operator installed prevents you from being able to install the latest version of Knative Eventing by using the {ServerlessOperatorName}.
```

Comment:
> Having the Knative Eventing Operator installed prevents you from being able to install the latest version of Knative Eventing by using the {ServerlessOperatorName}.

This sentence is throwing me a little. What do you think about rephrasing it a bit?

"If an earlier version of the Knative Eventing Operator is installed, you cannot install the latest version of the Operator using {ServerlessOperatorName}."

Reply:
Reworded this, thanks